src/containers/ArticleListing/index.js

```diff
@@ -97,17 +98,26 @@ export class ArticleListing extends React.Component {
           <div
             key={article && article.slug}
             className="slider-content"
-            style={{ background: `url('${(article && article.image) || placeholder}') no-repeat center center` }}
+            style={
```

Comment:
Avoid inline styles

Reply:
Thanks for the review @PaulKariukiRimiru 
 As you can see from this. The background image is dynamic (it is retrieved from every single article), in order for it to be displayed for each article. 
I could have used an <img> tag, but this won't work for the slider component. Kindly give me any suggestions you may have for this implementation.

Reply:
You can try and toggle use an `img` tag instead then you can toggle the image from the `src` prop, you can also apply styling to it as you did with the `div`

Reply:
I have implemented these changes @PaulKariukiRimiru